packages/xo-web/src/xo-app/backup-ng/new/metadata/index.js

```diff
@@ -169,6 +180,20 @@ export default decorate([
       setSettings: (_, _settings) => () => ({
         _settings,
       }),
+      setGlobalSettings: ({ setSettings }, name, value) => ({
+        settings = {},
+      }) => {
+        setSettings({
```

Comment:
first parameter it's an effect.

Reply:
I say that the first parameter of `setSettings`  is an effect not a setting name.